src/main/java/net/pms/dlna/LibMediaInfoParser.java

```diff
@@ -628,11 +632,10 @@ private static void getFormat(StreamType streamType, DLNAMediaInfo media, DLNAMe
 			format = FormatConfiguration.WAVPACK;
 		} else if (value.contains("mlp")) {
 			format = FormatConfiguration.MLP;
+		} else if (value.equals("openmg")) {
+			format = FormatConfiguration.ATRAC;
 		} else if (value.contains("atrac")) {
 			format = FormatConfiguration.ATRAC;
-			if (media.getCodecV() == null && !media.getContainer().equals(FormatConfiguration.RM)) {
-				media.setContainer(FormatConfiguration.ATRAC);
-			}
 		} else if (value.equals("jpeg")) {
```

Comment:
Sorry, i thought it was your code.
But that doesn't change the fact that it was not correctly handle, as other format, as WAV, having Atrac codec was considerate as Atrac container !
Anyway, do as you like. I'mm report error later, when you'll have merged :P